2.0.1/CHANGELOG.md

```diff
@@ -3,6 +3,4 @@
 Astronomer Certified 2.0.1-1 2020-01-21
```

Comment:
Also we haven't released it yet, so we should add `.dev` and not have a date.

Example: https://github.com/astronomer/ap-airflow/blob/master/2.0.0/CHANGELOG.md#astronomer-certified-200-3dev

Reply:
fixed in https://github.com/astronomer/ap-airflow/pull/202/commits/0015555238892b8349868d0501834ef544f806ed